IdentityCore/src/MSIDError.m

```diff
@@ -41,6 +41,7 @@
 NSString *MSIDKeychainErrorDomain = @"MSIDKeychainErrorDomain";
 NSString *MSIDHttpErrorCodeDomain = @"MSIDHttpErrorCodeDomain";
 NSString *MSIDInvalidTokenResultKey = @"MSIDInvalidTokenResultKey";
+NSInteger const MSIDSSOExtensionUnderlyingError = 2000;
```

Comment:
should we define it as something more unique? e.g. Apple might want to introduce next level of errors starting at 2000 as well.

Reply:
Changed to -6000.